src/main/groovy/wooga/gradle/paket/unity/PaketUpmPackageSpec.groovy

```diff
@@ -0,0 +1,68 @@
+package wooga.gradle.paket.unity
+
+import com.wooga.gradle.BaseSpec
+import org.gradle.api.provider.MapProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+
+trait PaketUpmPackageSpec implements BaseSpec {
+
+    /**
+     *
+     */
+    private final MapProperty<String, Map<String, Object>> paketUpmPackageJson = objects.mapProperty(String, Map)
+
+    @Input
+    @Optional
+    MapProperty<String, Map<String, Object>> getPaketUpmPackageJson() {
+        return paketUpmPackageJson
+    }
+
+    void setPaketUpmPackageJson(Map paketUpmPackages) {
+        this.paketUpmPackageJson.set(paketUpmPackages)
+    }
+
+    void setPaketUpmPackageJson(Provider<Map<String, String>> paketUpmPackages) {
+        this.paketUpmPackageJson.set(paketUpmPackages)
+    }
```

Comment:
Is this meant to be the contents of a package manifest? As someone familiar with all this I had to look it up elsewhere to realize this. I would update the name to reflect that.

Perhaps let's use the convention `packageManifest` for the map, and `packageManifestFile` that works off that.